ui/pages/settings/settings-search/settings-search.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SettingsSearch from './settings-search';
+
+export default {
+  title: 'SettingsSearch',
+  id: __filename,
+};
+
+export const SettingsSearchComponent = () => {
+  return <SettingsSearch />;
+};
```

Comment:
Story also seems to be broken. Run `yarn storybook` and go to http://localhost:6007/?path=/story/ui-pages-settings-settings-search-settings-search-stories-js--settings-search-component to see the component 

<img width="1440" alt="Screen Shot 2022-02-05 at 2 48 25 PM" src="https://user-images.githubusercontent.com/8112138/152628366-0a7f6dad-6c9e-4151-a49d-460dcbdaddbc.png">

Reply:
Done. 
You can see Search Box component in storybook now. 
Also if you want to use interactive Search Box you can do it through SettingsPage component in storybook.

Reply:
Awesome thanks @amerkadicE!!